<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>456</width>
    <height>282</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>456</width>
    <height>282</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>456</width>
    <height>282</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
    <weight>50</weight>
    <italic>false</italic>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>BaiduGaoWGS84_CoordinateTransform</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icon/globe.ico</normaloff>icon/globe.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Separation</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="commas_radBtn">
       <property name="text">
        <string>commas</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QRadioButton" name="tab_radBtn">
       <property name="text">
        <string>Tab</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="space_radBtn">
       <property name="text">
        <string>Blank Space</string>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Encoding</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="encoding_comboBox"/>
      </item>
      <item>
       <widget class="QCheckBox" name="skipHeadCheckBox">
        <property name="text">
         <string>First Line as Head</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Longitude</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="lngColBox">
          <property name="minimumSize">
           <size>
            <width>121</width>
            <height>22</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>From</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="fromCoordBox">
          <property name="minimumSize">
           <size>
            <width>121</width>
            <height>22</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Latitude</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="latColBox">
          <property name="minimumSize">
           <size>
            <width>111</width>
            <height>22</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>To</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="toCoordBox">
          <property name="minimumSize">
           <size>
            <width>111</width>
            <height>22</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Load File</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="2">
       <widget class="QLineEdit" name="fileLineEdit"/>
      </item>
      <item row="0" column="3">
       <widget class="QPushButton" name="loadFileBtn">
        <property name="text">
         <string>Open...</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Output File</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLineEdit" name="fileLineEdit_2"/>
      </item>
      <item row="1" column="3">
       <widget class="QPushButton" name="saveFileBtn">
        <property name="text">
         <string>Save...</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="runBtn">
      <property name="text">
       <string>Run</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionwhat_is_it"/>
   <addaction name="actionexit"/>
  </widget>
  <action name="actionwhat_is_it">
   <property name="icon">
    <iconset>
     <normaloff>icon/help.ico</normaloff>icon/help.ico</iconset>
   </property>
   <property name="text">
    <string>what is it?</string>
   </property>
  </action>
  <action name="actionexit">
   <property name="icon">
    <iconset>
     <normaloff>icon/close.png</normaloff>icon/close.png</iconset>
   </property>
   <property name="text">
    <string>exit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
